randovania/data/json_data/prime1.txt

```diff
@@ -3297,18 +3362,22 @@ Asset id: 3153742515
 > Dock to Fungal Hall B; Heals? False
   * Plasma Door to Fungal Hall B/Dock to Phazon Mining Tunnel
   > Dock to Fungal Hall A
-      All of the following:
-          Power Bomb and Morph Ball
-          Morph Ball Bomb or Boost Ball
+      Morph Ball Bomb and Power Bomb and Morph Ball
   > Pickup (Artifact of Newborn)
-      Morph Ball Bomb and Morph Ball and Phazon Suit
+      All of the following:
+          Morph Ball Bomb and Morph Ball
+          Any of the following:
+              Phazon Damage ≥ 2000
+              Boost Ball and Phazon Damage ≥ 1600
```

Comment:
Probably worth adding a trick for this, like movement. Also the damage values are too high. They both assume you need phazon suit. If you want one too require phazon, set it to damage 1500. The other I assume you don't, so you'd put 1400